src/services/jwt.js

```diff
@@ -0,0 +1,41 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '5s',
+  });
+
+  return token;
+}
+
+function verifyToken(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (error) {
+    return null;
+  }
+}
+
+function signRefresh(user) {
+  const token = jwt.sign(user, process.env.JWT_REFRESH_KEY, {
+    expiresIn: '30s',
```

Comment:
The expiration time for the refresh token is set to '30s', which is also very short. Typically, refresh tokens have a much longer expiration time than access tokens. Consider adjusting this to a longer duration.